conversion_utils/__main__.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+from enum import StrEnum
+from pathlib import Path
+
+import pandas as pd
+
+
+def convert_toetsingskader_to_csv(path: Path) -> None:
+    """
+    Converts the second tab in the toetsingskader algoritmes excel to
+    csv.
+    """
+    df: pd.DataFrame = pd.read_excel(path, sheet_name=1)
+    df.rename(columns=df.iloc[0], inplace=True)
+    df.drop(df.index[0], inplace=True)
+
+    out_dir = Path("input")
+    out_dir.mkdir(parents=True, exist_ok=True)
+    df.to_csv(out_dir / "toetsingskader_algoritmes_v1.csv", index=False)
```

Comment:
You could use the base stem (path.stem) of the incomming path variable as the output name and add the desired extension

Reply:
I agree this would be cleaner, but for now I left it as is, because the conversion script expects a file with a specific file name. Of course this can be mitigated but for now it's not important. The reason for this is, that the generated csv files only serve as an input for the yq transformations and can be discarted.